src/App.js

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 
 import './App.css';
-import Button from 'react-bootstrap/Button';
-import {Animated} from "react-animated-css";
+// import Button from 'react-bootstrap/Button';
```

Comment:
Let's remove these comments, in case we need to "remember" them we can just go back in history with Git :).